projects/storefrontlib/src/cms-components/checkout/components/shipping-address/address-form/address-form.component.ts

```diff
@@ -195,29 +198,24 @@ export class AddressFormComponent implements OnInit, OnDestroy {
   }
 
   verifyAddress(): void {
-    if (this.addressForm.valid) {
-      if (this.addressForm.get('region').value.isocode) {
-        this.regionsSub = this.regions$.pipe(take(1)).subscribe((regions) => {
-          const obj = regions.find(
-            (region) =>
-              region.isocode ===
-              this.addressForm.controls['region'].value.isocode
-          );
-          Object.assign(this.addressForm.value.region, {
-            isocodeShort: obj.isocodeShort,
-          });
+    if (this.addressForm.controls['region'].value.isocode) {
```

Comment:
Why did you change `this.addressForm.get('region').value.isocode` to `this.addressForm.controls['region'].value.isocode`? The get seems cleaner